luaui/Widgets/gui_info.lua

```diff
@@ -1248,22 +1239,11 @@ local function drawUnitInfo()
 		if unitDefInfo[displayUnitDefID].buildSpeed then
 			addTextInfo(texts.buildpower, unitDefInfo[displayUnitDefID].buildSpeed)
 		end
-		if unitDefInfo[displayUnitDefID].buildOptions then
-			addTextInfo(texts.buildoptions, #unitDefInfo[displayUnitDefID].buildOptions)
-		end
```

Comment:
This is just the number of build options a unit has, right? I agree this could be removed because we have icons for the unit's build options in there already... but that's only if that option is removed.

I'm not against this, just something to keep in mind :)